cpu/stm32f4/vectors.c

```diff
@@ -108,6 +108,10 @@ WEAK_DEFAULT void isr_otg_hs_ep1_in(void);
 WEAK_DEFAULT void isr_otg_hs_wkup(void);
 WEAK_DEFAULT void isr_otg_hs(void);
 WEAK_DEFAULT void isr_dcmi(void);
+WEAK_DEFAULT void isr_can3_tx(void);
```

Comment:
maybe protect with the same if defined as below or the build will fail for other boards (not sure).